### ISSUE TITLE
Can't run directly when running on a real machine

### ISSUE THREAD

**Issue by MyColourfulLife** [johnno1962/HotReloading#18]
It works well on the iOS simulator, 
but it can't run on the real machine. 
It reported the following error:
`dyld: Library not loaded: @rpath/SwiftTrace.framework/SwiftTrace
  Referenced from: /private/var/containers/Bundle/Application/99A71CEB-0F14-480C-9672-A7308B02F412/HotReloadTest.app/Frameworks/HotReloading.framework/HotReloading
  Reason: image not found
dyld: launch, loading dependent libraries
DYLD_LIBRARY_PATH=/usr/lib/system/introspection
DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib`
The framework is Already embed , I don't know how can run it successfully.
![image](https://user-images.githubusercontent.com/24448234/115987849-72fbce00-a5e9-11eb-80c8-c380a342d92c.png)

**Comment by johnno1962** [comment]
Is this Xcode 12.4? Check that SwiftTrace is included in the libraries to embed for the project. You can't use HotReloading on a device but you should still be able to run the app.

**Comment by MyColourfulLife** [comment]
YES,Xcode 12.4
<img width="998" alt="image" src="https://user-images.githubusercontent.com/24448234/115989074-126f8f80-a5ef-11eb-8010-a56299baf82e.png">
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/24448234/115989114-4d71c300-a5ef-11eb-8d81-1fad549ec7b2.png">
<img width="742" alt="image" src="https://user-images.githubusercontent.com/24448234/115989131-62e6ed00-a5ef-11eb-8f2d-50e3db64e99b.png">

**Comment by johnno1962** [comment]
In the project General properties in the image you showed above, click the + under "Frameworks, Libraries and Embedded content" and add the SwiftTrace framework. This is fixed in Xcode 12.5.

**Comment by MyColourfulLife** [comment]
OK，thank you very much and I will try it tomorrow.

**Comment by MyColourfulLife** [comment]
Thanks for the idea, the problem has been solved. I didn't try it on 14.5, I don't know if it works, because Xcode is too big and I didn't download the Xcode 12.5 RC version.
Actually, SwiftTrace Framework cannot be added directly. I added SwiftTrace as Swift Package Dependencies and it works.
<img width="692" alt="image" src="https://user-images.githubusercontent.com/24448234/116017241-a5e9a480-a671-11eb-900d-2d4b41827fb9.png">
